docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,10 @@
 Version history
 ---------------
 
+0.2.1-alpha1 (Jan 6, 2020)
+==========================
+This release contains a ref update for upstream Envoy. No changes to the feature set.
```

Comment:
This should have a newline after the `===` and I'd also mention the fixes that @kastiglione merged yesterday

Reply:
Wasn't that just a build fix?

Reply:
yes but I think it's probably worth noting since it could cause problems